api/.project/devstart.rb

```diff
@@ -0,0 +1,51 @@
+require_relative "common/common"
```

Comment:
I guess `.project` is a convention? I really don't like it; these are important scripts for using/running the project, so putting them in a (conventionally) hidden folder just seems to obscure how it's working. Is there a requirement that we put the scripts in a dot-folder?